OCaml/sudoku_solver/lib/sudoku.ml

```diff
@@ -0,0 +1,75 @@
+(* OCaml Version of the code *)
+
+module Sudoku = struct
```

Comment:
So you don't need to define a module here. Everything in this file `sudoku.ml` is compiled as a module accessible via the `Soduku` name. 

So having the
```ocaml
module Sudoku = struct
...
end
```
means that to access the functions inside the struct definition, you have to call `Sodoku.Sodoku.print_board`

Reply:
ahh okay nowander. This explains a lot thanks!